chapters/classes.tex

```diff
@@ -227,6 +227,8 @@ \subsubsection{Prefix Rules}\label{prefix-rules}
 Variables declared with the \lstinline!stream! type prefix shall be a subtype of \lstinline!Real!.
 This is further restricted in \cref{definition-of-stream-connectors}.
 
+Variables declared with the \lstinline!input! type prefix cannot also have the prefix \lstinline!parameter! or \lstinline!constant!.
```

Comment:
```suggestion
Variables declared with the \lstinline!input! type prefix cannot also have the prefix \lstinline!parameter! or \lstinline!constant!.
\begin{nonnormative}
Allowing the combination would require more complex rules for balancing models \cref{balanced-models}.
Parameters are not included in variable count, whereas inputs are counted as variables -- but balanced by counting their binding equations.
\end{nonnormative}
```
I did consider adding an explanation as above, but found that it confused more than it helped.